libs/src/services/ethContracts/serviceProviderFactoryClient.js

```diff
@@ -146,8 +146,8 @@ class ServiceProviderFactoryClient extends ContractClient {
     return info
   }
 
-  async getServiceProviderInfo (serviceType, serviceId) {
-    const method = await this.getMethod('getServiceProviderInfo',
+  async getServiceEndpointInfo (serviceType, serviceId) {
```

Comment:
shouldn't this become a GovernableContract?

Reply:
In my subsequent PR, it does become one! But not totally necessary for getting local setup back up :)